Sources/UINotificationCenter.swift

```diff
@@ -32,22 +32,29 @@ public final class UINotificationCenter {
     
     /// The window which will be placed on top of the application window.
     /// This window is used to present the notifications on and will be hidden when notifications are dismissed.
-    internal let window: UIWindow
+    internal lazy var window: UIWindow = {
+        let window: UIWindow
+        if let windowScene = UIApplication.shared.connectedScenes.filter({ $0.activationState == .foregroundActive }).first as? UIWindowScene {
```

Comment:
<!--
  2 failure:  'UIWindowScene' i..., 'connectedScenes'...
  1 warning:  Prefer using `.fi...
  
  
  DangerID: danger-id-Danger;
  File: Sources/UINotificationCenter.swift;
  Line: 37;
-->
- :no_entry_sign: 'UIWindowScene' is only available in iOS 13.0 or newer
- :no_entry_sign: 'connectedScenes' is only available in iOS 13.0 or newer
- :warning: Prefer using `.first(where:)` over `.filter { }.first` in collections. (`first_where`)